StRoot/StMuDSTMaker/COMMON/StMuFmsCollection.cxx

```diff
@@ -141,14 +141,18 @@ StMuFmsPoint* StMuFmsCollection::getPoint(int index) {
 }
 
 StMuFmsInfo* StMuFmsCollection::getInfo() {
-  if (!mInfo) return NULL;
-  return static_cast<StMuFmsInfo*>(mInfo->At(0));
+  if (!mInfo) return NULL;  
```

Comment:
Why use `NULL` here and `nullptr` below?

Reply:
I thought @plexoos suggested change for both so took it but indeed one remained NULL.